src/test/models/Offer.test.ts

```diff
@@ -0,0 +1,96 @@
+import { ApplicantType } from "$interfaces/Applicant";
+import { ApprovalStatus } from "$interfaces/ApprovalStatus";
+import { IPersistanceMyOffer, IPersistanceOffer } from "$interfaces/Offer";
+import { Secretary } from "$interfaces/Secretary";
+import { Offer } from "$models/Offer";
+
+describe("CurrentUser", () => {
+  const offerAttributes = {
+    uuid: "d3208f27-7404-4396-b8a1-88037d493989",
+    company: {
+      companyName: "Devartis",
+      cuit: "30711819017",
+      logo: "data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD",
+      uuid: "2005b62f-359b-4218-a673-e325fb4a06ef",
+      businessName: "businessName",
+      createdAt: "createdAt",
+      updatedAt: "updatedAt",
+      approvalStatus: "approvalStatus",
+      users: []
+    },
+    targetApplicantType: ApplicantType.both,
+    graduadosApprovalStatus: ApprovalStatus.approved,
+    extensionApprovalStatus: ApprovalStatus.approved,
+    graduatesExpirationDateTime: "2020-12-04T16:57:07.333Z",
+    studentsExpirationDateTime: "2020-12-04T16:57:07.333Z",
+    createdAt: "2020-12-04T16:57:07.333Z",
+    updatedAt: "2020-12-04T16:57:07.333Z",
+    careers: [],
+    sections: [],
+    title: "title",
+    description: "description",
+    hoursPerDay: 6,
+    isInternship: false,
+    minimumSalary: 10000,
+    maximumSalary: 10000,
+    _form: []
+  };
+
+  it("returns a valid Offer", () => {
+    const offer = Offer(offerAttributes as IPersistanceOffer);
+
+    expect(offer.isTargetToBoth()).toBe(true);
+    expect(offer.isTargetToGraduates()).toBe(false);
+    expect(offer.isTargetToStudents()).toBe(false);
+  });
+
+  it("returns a valid MyOffer", () => {
+    const offer = Offer<IPersistanceMyOffer>({
+      ...offerAttributes,
+      hasApplied: true
+    } as IPersistanceMyOffer);
+
+    expect(offer.hasApplied).toBe(true);
+  });
+
+  it("adds a method to ask if is target for Students", () => {
```

Comment:
Haria un test por metodo tipo asi: 
`returns false if the offer does not target students`